models/semantic_segmentation/DUC/cityscapes_loader.py

```diff
@@ -0,0 +1,167 @@
+import atexit
+import logging
+import multiprocessing as mp
+import random
```

Comment:
Add comments saying what these functions are doing or an overall comment for file

Reply:
done

Reply:
Please include some overall usage information back in the README on how and when you use this script.

Reply:
Added info about scripts in inference and validation sections of readme. Also added comments in the beginning of the scripts on where and what they are used for